index.html

```diff
@@ -27,6 +26,141 @@ <h1 class="section-header" id="my-name">Christian Onoh</h1>
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-ivan-samkov-5676744.jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 3</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-lukas-574070 (1).jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 4</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-mockupeditorcom-205316.jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 5</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-pixabay-39284.jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 6</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project"> See this project<i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>      
+    </section>  
+    <section class="section-title">
+      <header>
+        <h1 class="section-header">About</h1>
+      </header>
+      <p class="notes">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Corrupti in voluptates alias explicabo similique ab illum quas. Rerum minus assumenda adipisci possimus hic tenetur voluptatibus. Vitae laboriosam eveniet rerum quisquam.</p>
+      <p class="my-button"><span> Get my CV <i class="fa fa-download" aria-hidden="true"></i></span> </p>
```

Comment:
- [ ] I think it would be better also to use a link **(anchor tag)** ```a``` or ```button``` tag and remember to follow the button styling: **Active, Hover** and** Pressed for the best User experience and to replicate the Figma design as much as possible 🙏 

Kindly follow this [link](https://www.w3schools.com/howto/howto_css_animate_buttons.asp) for more on button interactions

**Figma button design**
![image](https://user-images.githubusercontent.com/102757126/219870938-bdbee47d-b652-46a5-b566-5cdecc4bf355.png)